docs/administration/projects/project-motd.md

```diff
@@ -5,21 +5,24 @@ The motd content is interpreted as [markdown](http://commonmark.org/help/) text
 
 ## Graphical Interface
 
-![Figure: Project motd](~@assets/img/fixme.png)
+![Figure: Project motd](~@assets/img/motdgui01.png)
+
+![Figure: Project motd](~@assets/img/motdgui02.png)
+
+![Figure: Project motd](~@assets/img/motdgui03.png)
 
 ## CLI Usage
 
 Update the motd with some text.
 
 ```bash
-rd projects motd put -p MyProject -t "This is the **motd** for MyProject"
+rd projects readme put -p MyProject --motd -t "This is the **motd** for MyProject"
 ```
 
 You might want to set the project home page to show the motd.
```

Comment:
Let's take out the "You might want to..."  That's poor documentation practice and change it to clearly explain what the next set of code is doing.